src/FileSystemUtilities.js

```diff
@@ -63,6 +64,9 @@ export default class FileSystemUtilities {
       }
       type = "file";
     }
+    if (pathIsAbsolute(src) && process.platform !== "win32") {
```

Comment:
Fair point, with the removing of the `type = 'dir'` earlier on, the point of having this around is getting weaker. I've gotten in the habit of checking for absolute paths as passing a relative path to `Path.relative` has always left me a bit confused:

```
$ node
> const path = require('path')
undefined
> path.relative('/wooo/aaa', '/wooo/bbb')
'../bbb'
> path.relative('/wooo/aaa', './bbb')
'../../Users/jaaprood/sites/lerna/bbb'
```

However, earlier on I made the argument that we can always expect an absolute path, so in this case those worries don't really mean much. I'll remove it, saving a dep 👍